mapbuilder/data/raw.py

```diff
@@ -0,0 +1,32 @@
+
+from pathlib import Path
+
+import logging
+import os
+
+class RAWParser:
+    result: dict | None
+
+    def __init__(self, file: Path):
+        self.result = None
+        self.file = file
+        
+    def parse(self):
+        result = {}
+        self.parse_recursively(self.file, result)
+        self.result = result
+        #logging.debug(result)
+        return self.result
+
+    def parse_recursively(self, root, result):
+        for dirname in os.listdir(root):
+            #logging.debug(f"1: {root / dirname}")
+            if os.path.isfile(root / dirname):
+                filepath = root / dirname
+                with (root / dirname).open(
+                    encoding="iso-8859-1",
+                ) as f:
+                    result[Path(*filepath.parts[1:]).as_posix()] = f.read()
+            elif os.path.isdir(root / dirname):
+                #logging.debug(f"2: {dirname}")
+                self.parse_recursively(root / dirname, result)
```

Comment:
What is the use case for this? Why is a single file und thus raw string per data item not enough?

Reply:
well, in FIR Langen they used an own script, to paste some data into maps. It worked simply by finding .import <path> and replacing it with txt files in the path. I know mapbuilder can import plain txt files, but I would need to get every single of these 100 txt files into mapbuilder.toml. I found it would be simpler if I could make the mapbuilder to search for the needed txt files by itself by parsing the whole directory with all these txt files and then in jinja provide a path to the file inside given directory. I couldn't just paste the data into maps, because I it was actually reused in completely separate maps for separate profiles.